src/backend/parser/parse_relation.c

```diff
@@ -485,6 +485,8 @@ GetRTEByRangeTablePosn(ParseState *pstate,
 					   int varno,
 					   int sublevels_up)
 {
+	Assert(pstate || sublevels_up == 0);
```

Comment:
```c
pstate = NULL;
sublevels_up = 0;
Assert(NULL || true);
...
return rt_fetch(varno, pstate->p_rtable); <--- segfault
```

Reply:
Seems like having sublevels_up == 0 doesn't have sense, thanks. Switched to Insist